README.md

```diff
@@ -1 +1,64 @@
-# amp-elixir
+# Amp Elixir
+
+[![Build Status](https://travis-ci.org/dwyl/amp-elixir.svg?branch=master)](https://travis-ci.org/dwyl/amp-elixir)
+[![codecov.io](https://codecov.io/github/dwyl/amp-elixir/coverage.svg?branch=master)](https://codecov.io/github/dwyl/amp?branch=master)
+[![Hex version](https://img.shields.io/hexpm/v/amp.svg "Hex version")](https://hex.pm/packages/amp)
+
+Amp transforms your Markdown into AMP-compliant html so it loads super-fast!
+
+## Why?
+
+Google is putting their clout behind the AMP (Accelerated Mobile Pages) Project to help speed up page load times on the Mobile Web: https://googleblog.blogspot.co.uk/2015/10/introducing-accelerated-mobile-pages.html
+
+This module lets you convert Markdown (`.md`) files to AMP-compliant `.html` so that you can serve them up to mobile clients much faster.
+
+## What?
+
+For many, reading on the mobile web is a slow, clunky and frustrating experience - but it doesn’t have to be that way. The Accelerated Mobile Pages (AMP) Project is an open source initiative that embodies the vision that publishers can create mobile optimized content once and have it load instantly everywhere.
```

Comment:
long lines are harder to update/review.
keep it to one sentence per line for readability.
http://rhodesmill.org/brandon/2012/one-sentence-per-line